packages/components/src/text-control/style.scss

```diff
@@ -27,6 +29,10 @@
 		padding-left: $grid-unit-15;
 		padding-right: $grid-unit-15;
 	}
+
+	&:focus {
+		@include input-style__focus( $components-color-accent);
+	}
```

Comment:
If we follow the changes suggested in https://github.com/WordPress/gutenberg/pull/69640/files#r2028695019, we may be able to remove these lines.